en/includes/guides/organization-management/organization-applications.md

```diff
@@ -0,0 +1,171 @@
+
+Please refer [Manage roles]({{base_path}}/guides/users/manage-roles/) for more details on how you can manage the roles.
+
+### Role types in organization
+
+Organization now have two types on roles.
+
+- Shared roles : Roles which are created in the organization when an application is shared with the particular organization.
+- Roles managed directly in the organization : Roles which can be managed directly in the organization.
+
+Both these roles can be assigned to any user in the organization but in the authorization, shared applications will only 
+consider the shared roles to provide the authorization.
+
+There might be scenarios where the above two types of roles can lead to a conflict of role creation. Please refer 
+[Manage role conflicts in organization]({{base_path}}/guides/organization-management/manage-conflicts-in-organizations/#manage-role-conflicts-in-organization) for more information.
+
+## OAuth2 Application Management
+
+Organizations can have their own OAuth2 applications and manage them independently. These applications can issue tokens 
+and allow access to the API Resources which belongs to the particular organization. Organization application can be 
+created with the below conditions.
+
+- Application's protocol is OAuth2.
+- Only client credentials, password and refresh grant types can be used.
+
+Organization application can be created by using the [Organization application Add]({{base_path}}/apis/organization-apis/organization-application-mgt/#tag/Applications/operation/createApplication) API request. A bearer token with the proper scopes 
+which was issued for the corresponding organization will be needed to invoke this API.
+
+These created applications can be edited from the Console application. 
+
+![Organization Application Edit]({{base_path}}/assets/img/guides/applications/organization-applications/organization-application-edit.png)
+
+Following operations are possible for organization applications.
+
+- Protocol level configurations
+- User attributes related configurations
+- API Authorization for organization application
+- Role management for organization application
+
+### Token generation from organization applications
+
+- Password Grant
+
+!!! abstract
+    **Request format**
+    ```
+    curl --user <OAUTH_CLIENT_KEY>:<OAUTH_CLIENT_SECRET> -k -d "grant_type=password&username=<USERNAME>&password=<PASSWORD>" -H "Content-Type: application/x-www-form-urlencoded" https://<IS_HOST>:<IS_PORT>/t/<TENANT_DOMAIN>/o/<ORG_ID>/oauth2/token
+    ```
+    ---
+    **Sample request**
+    ```curl
+    curl --user 7wYeybBGCVfLxPmS0z66WNMffyMa:WYfwHUsbsEvwtqmDLuaxF_VCQJwa -k -d "grant_type=password&username=Charlie&password=jG9A5KrX" -H "Content-Type: application/x-www-form-urlencoded" https://localhost:9443/t/carbon.super/o/7e98b86f-63c7-41a1-8c56-c909a21a2615/oauth2/token
```

Comment:
```suggestion
    curl --user 7wYeybBGCVfLxPmS0z66WNMffyMa:WYfwHUsbsEvwtqmDLuaxF_VCQJwa -k -d "grant_type=password&username=Charlie&password=jG9A5KrX" -H "Content-Type: application/x-www-form-urlencoded" https://localhost:9443/t/carbon.super/o/7e98b86f-63c7-41a1-8c56-c909a21a2615/oauth2/token
```

please template the hostname part and root org name part to be changed for IS and Asgardeo separately.
Update other places as well

Reply:
Done